docker/docs/builder/Dockerfile

```diff
@@ -6,10 +6,22 @@ ARG HTMLTEST_VERSION=0.17.0
 RUN CGO_ENABLED=0 go install github.com/wjdp/htmltest@v${HTMLTEST_VERSION} \
     && mv "${GOPATH}/bin/htmltest" /usr/bin/htmltest
 
+# we need clickhouse for running of the settings autogeneration script
+FROM clickhouse/clickhouse-server AS clickhouse-provider
```

Comment:
I think you want to use the [head](https://hub.docker.com/r/clickhouse/clickhouse-server/tags?name=head) image to have it at the latest.

But rather, maybe you want to use `https://github.com/ClickHouse/ClickHouse/blob/master/docs/_includes/install/universal.sh` to get the latest built binary for an appropriate arch+OS at the docs build time from `https://builds.clickhouse.com/master/${DIR}/clickhouse`